### PR TITLE
Update CS2D

### PR DESCRIPTION
**Pull request by ernestpasnik**
I've updated protocol and also changed the name to CS2D.
> Yes, CS2D was originally called "Counter-Strike 2D" because the initial idea was to make a 2D clone of Counter-Strike. In April 2017 however it was decided to rename it to "CS2D". With release 1.0.0.3 the name "CS2D" became official. From this time on Unreal Software is always referring to the game as "CS2D". The name "Counter-Strike 2D" is not used anymore.
https://cs2d.com/faq.php?show=misc_name#misc_name

### CODE REVIEW

**Comment by CosminPerRam** [comment]
Thanks a lot for contributing to gamedig!
Are there any places that documents the protocol changes? Maybe in a changelog or something? Also, dont they provide the number of bots anymore?

**Comment by ernestpasnik** [comment]
No, there are only game developer posts about a basic server query [https://www.unrealsoftware.de/forum_posts.php?post=427527&start=20#post427585](here).

I've reverse-engineered it to obtain these details. However, CS2D never returns information about who is a player and who is a bot. The previous code didn't always provide accurate information...

File: protocols/cs2d.js [hunk 1]
```diff
@@ -4,47 +4,41 @@ class Cs2d extends Core {
     async run(state) {
         {
             const reader = await this.sendQuery(
-                Buffer.from('\x01\x00\x03\x10\x21\xFB\x01\x75\x00', 'binary'),
-                Buffer.from('\x01\x00\xfb\x01', 'binary')
+                Buffer.from('\x01\x00\xFB\x01\xF5\x03\xFB\x05', 'binary'),
+                Buffer.from('\x01\x00\xFB\x01', 'binary')
             );
-            const flags = reader.uint(1);
-            state.raw.flags = flags;
```
Comment (CosminPerRam) [inline protocols/cs2d.js:11]: Although we do check for flags, someone might still want the data, please add this back.

**Comment by CosminPerRam** [comment]
Everything seems good, just that little detail and we can merge!

**Comment by ernestpasnik** [comment]
Alright, done! 😊

**Pull request merged**